ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionSimulator.java

```diff
@@ -110,7 +114,7 @@ public PrivateTransactionSimulator(
       // the following if.
     }
     if (privacyGroup == null || !privacyGroup.getMembers().contains(enclaveKey)) {
-      return privacyGroupDoesNotExistResult();
+      // The privacy group is on chain
```

Comment:
I have made some changes in this class (on upstream master) to make the priv_call work as the other priv RPCs with regards to multi tenancy (group membership). Membership is now checked in the PrivacyController.